tests/cpptests/test_cpp_expr.cpp

```diff
@@ -401,3 +401,241 @@ TEST_F(ExprTest, testPropertyFetch) {
   RLookupRow_Cleanup(&rr);
   RLookup_Cleanup(&lk);
 }
+
+
+
+TEST_F(ExprTest, testEvalFuncCase) {
+  TEvalCtx ctx;
+
+  // Basic case function tests - condition evaluates to true
+  ctx.assign("case(1, 42, 99)");
+  ASSERT_TRUE(ctx) << ctx.error();
+  ASSERT_EQ(EXPR_EVAL_OK, ctx.eval());
+  auto res = RSValue_Dereference(&ctx.result());
+  ASSERT_EQ(RSValue_Number, res->t);
```

Comment:
Consider creating a macro/function for this - repeats heavily

Reply:
➕ 1️⃣

Reply:
OK. Done. Thanks.